recipe/meta.yaml

```diff
@@ -34,32 +35,21 @@ test:
   imports:
     - importlib_resources
 
-outputs:
-  - name: importlib_resources
-  - name: importlib-resources
```

Comment:
We should keep both. The real package on PyPI is `importlib-resources`.

Reply:
Yep, the previous state of the recipe was fine https://github.com/AnacondaRecipes/importlib_resources-feedstock/blob/main/recipe/meta.yaml